### DISCUSSION TITLE
Complete beginner succeeds!

### DISCUSSION THREAD

**Discussion by tony1tf** [cardonabits/haxo-hw#57] [Show and tell]
Hi all

My Haxophone arrived a couple of weeks ago, but my wife confiscated the box and wrapped it up for Christmas. So, Christmas day, I had a spare R Pi Zero W, and one spare 16GB SD card (ps why is your image so big - isn't 8GB enough?). I couldn't get WiFi and SSH to work, but fortunately, the image booted up OK and the instrument started working. My only musical ability is poor guitar playing and a harmonica. I tried to download the video of you playing in the readme file, so I could play it in slow motion to learn  the keys, but wasn't able to. I looked at an on-line Sax course which helped a little.  However, we had a friend over for Christmas who is brilliant at improvising on the piano, and was able to play in a minor key that matched the haxophone. So after 3 days of alcohol fuelled practice, we produced this video of me playing  randomly along with his piano backing.

Tony

https://photos.app.goo.gl/uTn46uvGXXRLGbei8

**Comment by jcard0na** [comment]
Thank you for sharing, Tony!  You do not sound as a musical beginner... the alcohol fuelled practice seems to have worked!
I loved the good vibes in your video.

Stay musical!

PS.  Yes, I suppose I could use an 8GB image.  It is just that I have lots of 16GB SD cards... 😅

**Reply by tony1tf** [reply]
I have several 8GB cards, and that was my only spare 16GB one. Presumably there must be a program somewhere that can convert a 16GB image to an 8GB one as long as the extra is only empty memory.
